devops.yml

```diff
@@ -14,7 +14,7 @@ jobs:
 - job:
   displayName: Build project
   pool:
-    vmImage: windows-2019      # Name: 'Hosted Windows 2019 with VS 2019'
+    vmImage: windows-2022      # Name: 'Hosted Windows 2019 with VS 2019'
```

Comment:
Be careful with this, maybe the Nuget has problems when is used in other Services